todo/todo.css

```
@@ -0,0 +1,160 @@
+@import "reset.css";
+
+/* 화면 비율 설정 */
+@media (min-width: 768px) {
+  body {
+    /*웹앱 화면 설정하는 거 더 공부하기*/
+    width: 412px;
+    height: 100vh;
+    margin: 0 auto; /*이걸로 화면 가운데 정렬 했는데 이유는 더 찾아보기*/
+    background-color: lemonchiffon;
+  }
+}
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+header {
+  position: sticky;
+  background-color: lemonchiffon;
+  top: 0;
+  padding: 20px 15px;
+  width: 100%;
+  font-weight: bold;
+  font-size: 50px;
+  color: coral;
+  z-index: 1;
+}
+
+header > h1 {
+  text-align: center;
+}
+
+/* 달력 부분 */
+.weekly-section {
+  display: grid;
+  width: 412px;
+  /*그리드 가로 자꾸 스크롤 되면서 길어져서 일단 고정값 부여*/
+  height: 30vh;
+  grid-template-columns: repeat(7, 1fr);
+  row-gap: 10px;
+  column-gap: 10px;
+  justify-items: center;
+  align-items: center;
+  background-color: white;
+  border-radius: 10px;
+  padding: 0px 5px;
+  margin: 5px;
+  text-align: center;
+}
+
+.weekly-section__item {
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 0px;
+}
+
+.weekly-section__item__to-do-num {
+  position: relative;
+  margin-top: 30px;
+}
+
+.weekly-section__item__to-do-num__heart {
+  width: 30px;
+  height: 30px;
+}
+
+/* 하트랑 글자 겹쳐주기 */
+.weekly-section__item__to-do-num span {
+  position: absolute;
+  top: 22%;
+  left: 30%;
+  font-size: 18px;
+  color: white;
+}
+
+.weekly-section__item__date {
+  display: inline-block;
+  margin-top: 30px;
+}
+
+/* 투두리스트 부분 */
+.to-do {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  overflow-x: auto;
+  height: 75vh; /*높이 설정을 해줘야 overflow 적용 가능*/
+  margin: 5px;
+}
+
+.to-do__item {
+  margin: 20px 20px;
+}
+
+.to-do__item__title {
+  width: fit-content;
+  font-size: 23px;
+  padding: 10px;
+  font-weight: bold;
+  color: lemonchiffon;
+  border-radius: 15px;
+}
+/* nth-child 써서 배경색 다르게 적용하고 싶은데 잘못 쓰고 있는지 이상해져서 하나씩 적용했다,, */
+#to-do__first {
+  background-color: skyblue;
+}
+
+#to-do__second {
+  background-color: coral;
+}
+
+#to-do__third {
+  background-color: gold;
+}
+
+#to-do__fourth {
+  background-color: plum;
+}
```

```suggestion
.to-do__item:nth-of-type(1) h3 {
  background-color: skyblue;
}

.to-do__item:nth-of-type(2) h3 {
  background-color: coral;
}

.to-do__item:nth-of-type(3) h3 {
  background-color: gold;
}

.to-do__item:nth-of-type(4) h3 {
  background-color: plum;
}
```
요거 이렇게 바꾸면 된당! 
내 생각에 언니가 적용하면서 nth-child 지정 방식을 혼돈해서 적용이 잘 안된 것 같은데 [nth-child, nth-of-type css3 가상선택자 사용법 총정리](https://1004web.tistory.com/99) 이거 한 번 보고 공부해보면 좋겠다!!